src/test/java/sirius/db/mixing/FieldLookupCacheSpec.groovy

```diff
@@ -0,0 +1,74 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.db.mixing
+
+import sirius.db.jdbc.OMA
+import sirius.db.jdbc.SmartQueryTestEntity
+import sirius.db.mongo.Mango
+import sirius.db.mongo.MangoTestEntity
+import sirius.kernel.BaseSpecification
+import sirius.kernel.di.std.Part
+
+import java.time.LocalDateTime
+
+class FieldLookupCacheSpec extends BaseSpecification {
```

Comment:
maybe add a test with:
- a field in a composite
- a field in a mixin
- a composite field in a mixin

these are common problems when working with transformFromDatabase etc. as it is quite easy to get the accessPath wrong....